src/scrot_selection.c

```diff
@@ -92,6 +92,7 @@ static void createCursors(void)
 static void freeCursors(void)
 {
     struct Selection *const sel = *selectionGet();
+    scrotAssert(sel); /* silence clang-tidy */
```

Comment:
This one might warrant a bit more investigation. But it seemed like an FP to me.